freqtrade/exchange/exchange.py

```diff
@@ -2223,6 +2223,9 @@ async def _async_fetch_trades(self, pair: str,
                 )
                 trades = await self._api_async.fetch_trades(pair, since=since, limit=1000)
             trades = self._trades_contracts_to_amount(trades)
+
+            if len(trades) > 0:
+                logger.debug( "Fetched trades for pair %s, datetime: %s (%d).", pair, trades[0]['datetime'],  trades[0]['timestamp']  )
```

Comment:
not sure what this is fixing ... ? 
it's just adding a conditional log statement - but won't change the return value or whatever ... ?

Reply:
It's fixing a crash via exception when trades are empty but `logger.debug` access them via `trades[0]`, you see?

Reply:
That's correct - but considering the diff - both the if, and the log message are new (so there's currently no debug output - or at least not in this area of the code).

therefore I'm not sure what this intends to fix, as it's only adding lines, but not replacing anything.

Reply:
> but considering the diff - both the if, and the log message are new (so there's currently no debug output -

oh, that's what you mean. I'm sorry, I must have written this `logger.debug` statement a few months ago myself. I thought it was original freqtrade code.